### ISSUE TITLE
Ability to load tag from a file

### ISSUE THREAD

**Issue by max-zelinski** [vallard/drone-kube#4]
I would be great if we can use a dynamically generated tag (for instance constructed from package.json version, date, and moon phase) in `drone-kube`. The tag can be constructed and saved to a file so `drone-kube` can use it.
This functionality is already supported by `drone-docker` (it looks for `.tag` file).

**Comment by jdemaris** [comment]
What are you using to deploy the other components? For example, I wanted to create an `Ingress` inside of this file as well, but attempting to do that, even in its own dedicated file, results in:

```
2018/07/09 14:11:52 Error decoding yaml file to jsonconverting (v1.Service).v1.ServiceSpec to (v1beta1.Deployment).v1beta1.DeploymentSpec: Replicas not present in src
```

**Comment by vallard** [comment]
Yeah we would have to add that. It only does deployments right now. It could probably be modified to do that but not something I have time for now. Those other parts (service, ingress) were all static in my setup so the only thing that really changed was the deployment file with different versions of the container image.
